extension/src/NABfunctions.ts

```diff
@@ -217,11 +218,18 @@ export async function findNextUnTranslatedText(
         );
       }
     }
+    if (nextUntranslated) {
+      DocumentFunctions.openTextFileWithSelection(
+        nextUntranslated.filePath,
+        nextUntranslated.position,
+        nextUntranslated.length
+      );
+    }
   } catch (error) {
     showErrorAndLog("Find next untranslated", error as Error);
     return;
   }
-  if (!foundAnything) {
+  if (!nextUntranslated) {
     vscode.window.showInformationMessage(`No more untranslated texts found.`);
   }
   console.log("Done: FindNextUnTranslatedText");
```

Comment:
In quite a few places I've written "UnTranslated" with a capital T, but I really think that it should be a lowercase t... Maybe it's time to fix this?

Reply:
fixed :)